### DISCUSSION TITLE
microsoft/dotnet latest tag updated to .NET Core 2.0

### DISCUSSION THREAD

**Discussion by richlander** [dotnet/dotnet-docker#3644] [Announcements]
# microsoft/dotnet `latest` tag updated to .NET Core 2.0 SDK

The `latest` tag for the [microsoft/dotnet](https://hub.docker.com/r/microsoft/dotnet/) repo has been updated to reference the [.NET Core 2.0](https://blogs.msdn.microsoft.com/dotnet/2017/08/14/announcing-net-core-2-0) SDK.

## Details

* [dotnet-docker #277](https://github.com/dotnet/dotnet-docker/pull/277)
* [aspnet-docker #278](https://github.com/aspnet/aspnet-docker/pull/278)

The `latest` tag for the [microsoft/dotnet](https://hub.docker.com/r/microsoft/dotnet/) repo is always intended to reference the latest supported .NET Core SDK. At this time, the latest support SDK is .NET Core SDK 2.0.

The following repos have been updated in the same way:

* [microsoft/dotnet](https://hub.docker.com/r/microsoft/dotnet/)
* [microsoft/aspnetcore](https://hub.docker.com/r/microsoft/aspnetcore/)
* [microsoft/aspnetcore-build/](https://hub.docker.com/r/microsoft/aspnetcore-build/)
* [microsoft/dotnet-samples/](https://hub.docker.com/r/microsoft/dotnet-samples/)

It is recommended to use a more-specific tag than `latest` more anything other than casual development. Once you have an application that you believe will eventually go to production, you are recommended to use a tag like `2.0-sdk`. The [.NET Core Docker Samples](https://github.com/dotnet/dotnet-docker-samples) contains examples of more specific usage such as the [.NET Core Development Sample Dockerfile](https://github.com/dotnet/dotnet-docker-samples/blob/master/dotnetapp-dev/Dockerfile).

Also published at [dotnet/announcements #28](https://github.com/dotnet/announcements/issues/28).

**Comment by dasMulli** [comment]
Should the "old" 1.* images be updated to the 1.1.0 SDK to match VS 2017 15.3?

**Comment by MichaelSimons** [comment]
@dasMulli - Thanks for pointing this out.  Yes the 1.* images should have been updated.  This has been fixed.  Please re-pull.
